project/api/models/civi.py

```diff
@@ -163,7 +169,7 @@ def score(self, request_acct_id=None):
         f = 0 #TODO: favorite val, is 'favorite' a meaningful name for this variable?
 
         # Calculate how long ago the post was created
-        time_ago = (current_time - post_time.replace(tzinfo=None)).total_seconds() / 300
+        time_ago = old_div((current_time - post_time.replace(tzinfo=None)).total_seconds(), 300)
```

Comment:
I dont think we have any integer division we want to keep in the score function so you can revert all the `old_div` calls back to `/`